health/health.d/beanstalkd.conf

```diff
@@ -11,9 +11,11 @@ component: Beanstalk
      warn: $this > 0
      crit: $this > 10
     delay: up 0 down 5m multiplier 1.2 max 1h
-     info: number of buried jobs across all tubes. \
-           You need to manually kick them so they can be processed. \
-           Presence of buried jobs in a tube does not affect new jobs.
+     info: Number of buried jobs across all tubes. \
```

Comment:
@ilyam8 we could elaborate on Beanstalk and even link to its documentation

Reply:
The details and links will be in the Beanstalk charts descriptions and alarms guides (which the alarm will have a link to).